lib/bot.js

```diff
@@ -208,6 +208,11 @@ class Bot {
         this.ircClient.say(ircChannel, prelude);
       } else if (!message.subtype) {
         text = `<${user.name}> ${text}`;
+      } else if (message.subtype === 'file_share') {
+        text = `<${user.name}> File uploaded ${message.file.permalink} / ${message.file.permalink_public}`;
```

Comment:
Isn't showing one of these enough? I.e. `message.file.permalink_public || message.file.permalink`.